packages/vite-plugin/vitest.config.ts

```diff
@@ -0,0 +1,23 @@
+import replace from "@rollup/plugin-replace";
+import { defineProject } from "vitest/config";
+import { config, plugins } from "../../vitest.shared";
+
+const packageJson = await import("./package.json");
+
+export default defineProject({
+  ...config,
+  plugins: [
+    // @ts-expect-error - using rollup plugin
```

Comment:
What is the TS error? Doing `import * as replace` doesn't squash it?

Reply:
Gives me a `This expression is not callable` error 🤔

Reply:
```
packages/vite-plugin test:unit$ vitest run
│ ▲ [WARNING] Calling "replace" will crash at run-time because it's an import namespace object, not a function [call-import-namespace]
│     vitest.config.ts:11:9:
│       11 │       ...replace({
│          ╵          ~~~~~~~
│   Consider changing "replace" to a default import instead:
│     vitest.config.ts:1:429:
│       1 │ .../vitest.config.ts";import * as replace from "@rollup/plugin-repl...
│         │                              ~~~~~~~~~~~~
│         ╵                              replace
│   Make sure to enable TypeScript's "esModuleInterop" setting so that TypeScript's type checker generates an error when you try to do this. You can read more about this setting here: https://www.typescriptlang.org/tsconfig#esModuleInterop
│ failed to load config from /Users/<user>/Projects/codecov-javascript-bundler-plugins/packages/vite-plugin/vitest.config.ts
```

Reply:
I think we need to turn on `esModuleInterop`

Reply:
I do have it currently turned on in the root config:

https://github.com/codecov/codecov-javascript-bundler-plugins/blob/main/tsconfig.json#L34

Reply:
okay weird - it should work with `import replace from ...`. ESM causing issues again 😭

Reply:
When I remove the expect this is the error i get :sob:

```
Type '{ enforce: "pre"; api?: any; cacheKey?: string | undefined; name: string; version?: string | undefined; augmentChunkHash?: ObjectHook<(this: PluginContext, chunk: RenderedChunk) => string | void, {}> | undefined; ... 25 more ...; watchChange?: ObjectHook<...> | undefined; }' is not assignable to type 'PluginOption'.
  Type '{ enforce: "pre"; api?: any; cacheKey?: string | undefined; name: string; version?: string | undefined; augmentChunkHash?: ObjectHook<(this: PluginContext, chunk: RenderedChunk) => string | void, {}> | undefined; ... 25 more ...; watchChange?: ObjectHook<...> | undefined; }' is not assignable to type 'undefined'.
```